tech-jobs/summary_stats/scripts/update_edu_dpmt.py

```diff
@@ -0,0 +1,68 @@
+import re
+import sys
+import os
+import unicodecsv as csv
+import pandas as pd
+import itertools
+import numpy as np
+from collections import Counter
+
+import science.roles as r
+roles = r.Roles()
+
+empl_path = sys.argv[1] #directory of csv files to process
+#target = sys.argv[2] #csv file name to write
```

Comment:
The new data you will work with already has the edu and department code implemented here, so you should be able to simplify and remove this for json input files. Keep it for the existing files, since we don't want to lose the ability to work with them.